modules/core/02-client/migrations/v7/store_test.go

```diff
@@ -40,23 +36,6 @@ func TestIBCTestSuite(t *testing.T) {
 	suite.Run(t, new(MigrationsV7TestSuite))
 }
 
-// test that MigrateStore returns an error if the codec used doesn't register tendermint types
-func (suite *MigrationsV7TestSuite) TestMigrateStoreTendermint() {
```

Comment:
`GetClientState` uses the client keeper codec which has tendermint client types registered so this test is no longer useful